src/pages/iou/WaypointEditor.tsx

```diff
@@ -127,9 +109,9 @@ function WaypointEditor({route: {params: {iouType = '', transactionID = '', wayp
         return errors;
     };
 
-    const saveWaypoint = (waypoint) => Transaction.saveWaypoint(transactionID, waypointIndex, waypoint, isEditingWaypoint);
+    const saveWaypoint = (waypoint: OnyxTypes.RecentWaypoint) => Transaction.saveWaypoint(transactionID, waypointIndex, waypoint, isEditingWaypoint);
 
-    const submit = (values) => {
+    const submit = (values: Record<string, string>) => {
```

Comment:
Same for `validate` comment, `onSubmit` prop is called with result of `ValidationUtils.prepareValues()`

Reply:
same answer 😄